app/public/cantusdata/management/commands/import_data.py

```
@@ -8,12 +8,32 @@
 from cantusdata.helpers.source_importer import SourceImporter
 import urllib.request
 import urllib
+from urllib.parse import urlsplit, urlunsplit
 from io import StringIO
 from cantusdata.settings import BASE_DIR
 from os import path
 import csv
 
 
+def _normalize_scheme_to_https(url):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_normalize_scheme_to_https`

(ANN202)